Makefile

```diff
@@ -25,33 +31,33 @@ help: ## Prints help for targets with comments
 	@grep -E '^[a-zA-Z0-9.\ _-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
 
 build:
-	@docker build -t $(IMAGE_NAME):latest -f Dockerfile .
-	@docker tag $(IMAGE_NAME):latest $(IMAGE_NAME):$(IMAGE_TAG)
+	@$(CONTAINER_ENGINE) build -t $(IMAGE_NAME):latest -f Dockerfile .
+	@$(CONTAINER_ENGINE) tag $(IMAGE_NAME):latest $(IMAGE_NAME):$(IMAGE_TAG)
 
 push:
-	@docker --config=$(DOCKER_CONF) push $(IMAGE_NAME):latest
-	@docker --config=$(DOCKER_CONF) push $(IMAGE_NAME):$(IMAGE_TAG)
+	@$(CONTAINER_ENGINE) --config=$(DOCKER_CONF) push $(IMAGE_NAME):latest
+	@$(CONTAINER_ENGINE) --config=$(DOCKER_CONF) push $(IMAGE_NAME):$(IMAGE_TAG)
 
 bundle: ## Use qontract-validator image to bundle schemas into $BUNDLE_FILENAME NOTE
 	mkdir -p $(OUTPUT_DIR) fake_data fake_resources
 	@$(CONTAINER_ENGINE) run --rm \
-		-v $(PWD)/schemas:/schemas:z \
-		-v $(PWD)/graphql-schemas:/graphql:z \
-		-v $(PWD)/fake_data:/data:z \
-		-v $(PWD)/fake_resources:/resources:z \
+		-v $(PWD)/schemas:/schemas$(SELINUX_MOUNT_CHAR) \
+		-v $(PWD)/graphql-schemas:/graphql$(SELINUX_MOUNT_CHAR) \
+		-v $(PWD)/fake_data:/data$(SELINUX_MOUNT_CHAR) \
+		-v $(PWD)/fake_resources:/resources$(SELINUX_MOUNT_CHAR) \
```

Comment:
`:z` means to share with multiple containers, I don't see where are we using that feature, all volume binds in this file are just read only mount, a simple `:ro` should be good enough.

Reply:
@hemslo With regard to `:z`, while you are correct, the container engine facilitates the sharing through SELinux labels, which aren't available outside a SELinux-enabled environment (such as MacOS).

If we want to change the flag to `:ro` later, we can do that at a later date. I want this change to be minimal in scope without potentially breaking changes.

Reply:
Then can we only keep `@$(CONTAINER_ENGINE)` changes in this pr, or just remove `:z` in both cases? `:ro` is a good to have suffix to indicate readonly, then that system check is also not needed anymore, seems small enough to be included all in one change.